pyproject.toml

```
@@ -22,6 +22,8 @@ include = ["sas7bdat_converter/py.typed"]
 python = "^3.8.0"
 pandas = ">=1.2.0"
 openpyxl = {version = ">=3.0.5", optional = true}
+pyarrow = {version = ">=9.0.0", python = ">=3.7, <=3.10", optional = true}
+fastparquet = {version = ">=0.8.3", python = ">=3.11", optional = true}
```

Lets leave `pyaarow` as optional and only use it to keep things simple. I am sure they will do a release for 3.11 soon so I'm not worried about the failing CI tests for that right now.

```suggestion
pyarrow = {version = ">=9.0.0", optional = true}
```

With this we will also need to update the extras section below:

```toml
[tool.poetry.extras]
excel = ["openpyxl"]
parquet = ["pyarrow"]
all = ["openpyxl", "pyarrow"]
```